risc0/cargo-risczero/src/utils.rs

```diff
@@ -22,8 +22,13 @@ use std::{
 };
 
 use anyhow::{anyhow, Context, Result};
+use bonsai_sdk::{alpha::Client, API_KEY_ENVVAR, API_URL_ENVVAR};
+use clap::Args;
 use fs2::FileExt;
 
+/// Reports the current version of this crate.
+pub const VERSION: &str = env!("CARGO_PKG_VERSION");
```

Comment:
The version of `cargo-risczero` shouldn't be used for this. We should use the version of `risc0-zkvm`.

Reply:
Oh, I thought that by sharing the same workspace and version definition there wouldn't be a problem doing the same thing as in `risc0-zkvm`. But I changed no to use `risc0_zkvm::VERSION` instead ✅

Thanks for pointing out!